notebooks/tel_and_site/subsys_req_ver/m2/m2_functional_verification/LVV-T1782_M2_RMP_functionality_check_small_range.ipynb

```diff
@@ -0,0 +1,329 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.            ti = time.time()
</pre><p>In such condition, usually, you should use the monotonic time instead:</p><p><a href="https://medium.com/techtofreedom/5-built-in-timers-in-python-differences-and-usages-596c4c815457#:~:text=time.,-monotonic()%3A%20A&amp;text=monotonic()%20function%20are%20more,of%20two%20calls%20makes%20sense." target="_blank">https://medium.com/techtofreedom/5-built-in-timers-in-python-differences-and-usages-596c4c815457#:~:text=time.,-monotonic()%3A%20A&amp;text=monotonic()%20function%20are%20more,of%20two%20calls%20makes%20sense.</a></p><p> </p><p>You can name the variable to be **time_initial**, which is more clear.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>